core/templates/components/forms/custom-forms-directives/image-uploader.component.ts

```diff
@@ -95,9 +96,48 @@ export class ImageUploaderComponent {
     let filename: string = this.imageInputRef.nativeElement.value.split(
       /(\\|\/)/g).pop();
     this.errorMessage = this.validateUploadedFile(file, filename);
+    let fchng = this.fileChanged;
```

Comment:
What does fchng mean? Can you please give a more descriptive name that explains what it represents?